data/json/monsters/feral_humans.json

```diff
@@ -737,6 +737,7 @@
     "melee_dice_sides": 3,
     "dodge": 1,
     "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_syn_armored" ],
```

Comment:
Description says "body-armor **wasn't** one of their priorities". Why prof_wp_syn_armored?

Reply:
Ah you're right, I misparsed that lol.

Reply:
Fixed